defold-spine/commonsrc/vertices.cpp

```diff
@@ -221,6 +221,7 @@ uint32_t CalcVertexBufferSize(const spSkeleton* skeleton, spSkeletonClipping* sk
         {
             CalcAndAddVertexBufferAttachment(attachment, &vertex_count, &max_triangle_count);
         }
+        spSkeletonClipping_clipEnd(skeleton_clipper, slot);
```

Comment:
When we calculate buffer size, we should turn clipping off, otherwise vertex buffer size will be wrong